app/views/repository/_form.html.haml

```diff
@@ -1,27 +1,52 @@
 %form#form_div{:name => "repositoryForm", 'ng-controller' => "repositoryFormController"}
   = render :partial => "layouts/flash_msg"
-  %fieldset
-    %h3 Basic Information
-    %table.admintable
-      %tbody
-        %tr
-          %td.key Name
-          %td.wide
-            %input{:type => "text", :name => "repo_name", 'ng-model' => "repoModel.repo_name", 'ng-maxlength' => 10, :miqrequired => true, :checkchange => true}
-        %tr
-          %td.key Path
-          %td.wide
-            %input{:type => "text", :name => "repo_path", 'ng-model' => "repoModel.repo_path", 'ng-maxlength' => 100, :miqrequired => true, :checkpath => true, :checkchange => true}
+  %h3
+    = _('Basic Information')
+  .form-horizontal
+    .form-group
+      %label.col-md-2.control-label
+        = _('Name')
+      .col-md-8
+        %input.form-control{:type          => "text",
```

Comment:
@martinpovolny I've suggested @PanSpagetka to move the class from the hash out and it looks like he forgot to adjust the indentation